charmhelpers/core/hookenv.py

```diff
@@ -119,19 +121,19 @@ def log(message, level=None):
             raise
 
 
-def action_log(message):
-    """Write an action progress message"""
-    command = ['action-log']
+def function_log(message):
```

Comment:
do we need to keep `action_log`?

Reply:
no since it was only added during rc and is not used